bitcoin_transfer/spend_your_coin.md

```diff
@@ -177,38 +182,39 @@ transaction.Outputs.Add(new TxOut()
 });
 ```
 
-To sum up take a look at my whole transaction before we sign it:  
-I have 3 **TxOut**, 2 with **value**, 1 without **value** \(which contains the message\). You can notice the differences between the **scriptPubKey**s of the "normal" **TxOut**s and the **scriptPubKey** of the **TxOut** with the message:
+### Summary
+To sum up, let's take a look at the whole transaction before we sign it:  
+We have 3 **TxOut**, 2 with **value**, 1 without **value** \(which contains the message\). You can notice the differences between the **scriptPubKey**s of the "normal" **TxOut**s and the **scriptPubKey** of the **TxOut** within the message:
 
 ```json
 {
-  "hash": "b7803df4b90fd615532bcbdb3b63eb1af5a2e4ae36f29a6fbf9f57d0a1842e0a",
+  "hash": "579ae72016c02691a9e47258d65705575d0a6a5f265b137c67ef6be7b2a35659",
```

Comment:
Please pay attention for these to be consistent. I'm kind of confused during this review so I'm not sure I requested modification in the previous lines or not.

Reply:
That's again different because of the new transaction with the new values. It is consistent with the data in the block explorer.